app/src/main/java/com/blaisemuhirwa/covid_19trackerandroidapp/Home.java

```diff
@@ -61,6 +63,21 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         newDeathsTextView = view.findViewById(R.id.newDeathsTextView);
```

Comment:
Instead of using findViewById, we can use ViewBinding. It limits the search for the view, it is neat, and it's easy to write. [Check out view binding](https://guides.codepath.com/android/Applying-Data-Binding-for-Views).